iogt/templates/core/section_page.html

```diff
@@ -1,7 +1,7 @@
 
 {% extends "base.html" %}
 
-{% load static core_tags %}
+{% load static core_tags i18n %}
```

Comment:
I'm wondering why we're loading `i18n` here since I don't see any `trans` or `blocktrans` being used?